proof/invariant-abstract/ARM/ArchTcb_AI.thy

```diff
@@ -238,8 +237,9 @@ lemma tc_invs[Tcb_AI_asms]:
   apply (simp add: split_def set_mcpriority_def cong: option.case_cong)
   apply (rule hoare_vcg_precond_imp)
    apply wp
-      (* takes long: *)
-      apply ((simp only: simp_thms
+      (* takes long: (30 sec) *)
+      apply ((simp only: simp_thms cong: conj_cong
```

Comment:
very old style, might be worth reindenting the `|`, but not if you're just blending in

Reply:
actually, changed my mind, then you'd have to do the same on ARM_HYP, and it's probably not worth it